README.md

```diff
@@ -1,9 +1,9 @@
 # Swagger for Rails 5
 
-[![Build Status](https://travis-ci.org/ManageIQ/topological_inventory-api.svg)](https://travis-ci.org/ManageIQ/topological_inventory-api)
-[![Maintainability](https://api.codeclimate.com/v1/badges/47776e67dbb7cc572c3b/maintainability)](https://codeclimate.com/github/ManageIQ/topological_inventory-api/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/47776e67dbb7cc572c3b/test_coverage)](https://codeclimate.com/github/ManageIQ/topological_inventory-api/test_coverage)
-[![Security](https://hakiri.io/github/ManageIQ/topological_inventory-ingress_api/master.svg)](https://hakiri.io/github/ManageIQ/topological_inventory-ingress_api/master)
+[![Build Status](https://travis-ci.org/RedHatInsights/topological_inventory-api.svg)](https://travis-ci.org/RedHatInsights/topological_inventory-api)
+[![Maintainability](https://api.codeclimate.com/v1/badges/47776e67dbb7cc572c3b/maintainability)](https://codeclimate.com/github/RedHatInsights/topological_inventory-api/maintainability)
```

Comment:
Same like in core, codeclimate's url is not moved to RedHatInsights.

Also it seems there is a bad definition: `topological_inventory-api` instead of `topological_inventory-ingress_api`

Reply:
@slemrmartin are these changes still needed for this PR? Is there value in opening a new one if we need to get them in yet?

Reply:
@syncrou these changes were implemented by @gmcculloug few days ago. Closing this PR

Reply:
@gtanzillo ^^